plugin/pkg/scheduler/factory/factory_test.go

```diff
@@ -63,6 +63,8 @@ func TestCreate(t *testing.T) {
 		informerFactory.Apps().V1beta1().StatefulSets(),
 		informerFactory.Core().V1().Services(),
 		v1.DefaultHardPodAffinitySymmetricWeight,
+		// enable equivalence class cache
```

Comment:
Yes, but do you think it reasonable enable this in tests? (or, it's a little bit dangerous for 1.7)